YIF.Core.Domain/ApiModels/ResponseApiModels/IoEforSuperAdminResponseApiModel.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Text.Json.Serialization;
+using YIF.Core.Data.Entities;
+
+namespace YIF.Core.Domain.ApiModels.ResponseApiModels
+{
+    public class IoEforSuperAdminResponseApiModel
+    {
+        /// <summary>
+        /// Unique id
+        /// </summary>
+        /// <example>e2bd4ad9-060b-4d53-8222-9f3e5efbcfc7</example>
+        public string Id { get; set; }
```

Comment:
i think that name of this field could be specified to IoEId

Reply:
done=)